dali/operators/image/remap/warp_affine_params.h

```diff
@@ -67,7 +73,7 @@ class WarpAffineParamProvider
       for (int i = 0; i < spatial_ndim; i++)
         for (int j = 0; j < spatial_ndim+1; j++, k++)
           M.transform(i, j) = matrix[k];
-
+      if (invert) M = M.invert();
       auto *params = this->AllocParams(kernels::AllocType::Host);
       for (int i = 0; i < num_samples_; i++)
         params[i] = M;
```

Comment:
mmm, how do you access params later?

Reply:
AllocParams allocates memory, assignes it to `params_cpu_` or `params_gpu_` and returns the pointer to it. So later I access the params with `params_[gpu/cpu]_`